### ISSUE TITLE
Remove hardcoding of start date of semester

### ISSUE THREAD

**Issue by banool** [banool/lectureDL#5]
Currently each the start and end dates of semester are hardcoded into the program, and need to be changed each semester. The same goes for when the mid semester break is.

It would be nice to have some way to configure this info via settings, or perhaps even to not need it at all (by just inferring the start date from the LMS / Echocenter somehow).

Probably depends on #4.
